pytorch_lightning/callbacks/progress/rich_progress.py

```diff
@@ -190,6 +190,23 @@ class RichProgressBarTheme:
     metrics: Union[str, Style] = "white"
 
 
+def _detect_light_colab_theme() -> bool:
```

Comment:
shouldn't this function be defined at the bottom of the file?

Reply:
Updated!